core-java-modules/core-java/src/main/java/com/baeldung/clone/ShallowMatch.java

```diff
@@ -0,0 +1,30 @@
+package com.baeldung.clone;
+
+import java.time.OffsetDateTime;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@AllArgsConstructor
+@Data
+public class ShallowMatch implements Cloneable {
+
+    private TYPE type;
+
+    private OffsetDateTime startTime;
+
+    private OffsetDateTime endTime;
+
+    private Integer durationInMinutes;
+
+    private PlayersDetail playersDetail;
+
+    public enum TYPE {
```

Comment:
why not just `Type` (CamelCase)
It's the standard, I think, see: https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html

Reply:
Thanks for your comments, done